README.md

```diff
@@ -13,3 +13,47 @@ Once you've done that, check out:
 - [Q&A](https://github.com/MonacoProtocol/sdk/discussions/categories/q-a)
 
 Any troubles, issues, questions, ideas you have - anything - raise them in discussions and the community will help you out.
+
+# Introduction
+
+The Monaco Protocol is a decentralized liquidity network for exchange-based applications built on the Solana blockchain. It serves as crucial infrastructure upon which third-party front-end applications for wagering on binary outcome events can be built while accessing a shared liquidity pool.
```

Comment:
Given all the glossary discussions recently, we should make sure `wagering` is the right verb to use here.

Reply:
(and throughout)

Reply:
yeah 😬

Reply:
can probably stick with it for now though then change it every week :P